Tests/SwiftParserTest/translated/InvalidTests.swift

```diff
@@ -289,29 +256,56 @@ final class InvalidTests: XCTestCase {
     )
   }
 
-  func testInvalid16() {
+  func testInvalid16a() {
     AssertParse(
       """
       func f1_43591(a : inout inout Int) {}
-      func f2_43591(inout inout b1️⃣: Int) {}
-      func f3_43591(let let 2️⃣a: Int) {}
-      func f4_43591(inout x3️⃣: inout String) {}
-      func f5_43591(inout i4️⃣: inout Int) {}
+      """
+    )
+  }
+
+
+  func testInvalid16b() {
+    AssertParse(
+      """
+      func f2_43591(1️⃣inout inout b: Int) {}
+      """,
+      diagnostics: [
+        DiagnosticSpec(message: "'inout inout' before a parameter name is not allowed", fixIts: ["move 'inout inout' in front of type"]),
+      ], fixedSource: "func f2_43591(b: inout Int) {}"
```

Comment:
That's somewhat confusing 😅. What happens when there's already an `inout` in this case? Does it say "remove redundant inout inout"?